docs/tutorials/single-dev-workflow.md

```diff
@@ -0,0 +1,206 @@
+---
+sidebar_label: "Epinio Journey: Single Dev Workflow"
+title: ""
+---
+
+## Introduction
+
+While the [quickstart](./quickstart.md) is great to get started, as a Developer, you might want to see end-to-end solutions and how Epinio can help you.
+That's exactly the aim of the "Epinio Journeys", where you'll be able to follow different use-cases, according to your needs.
+
+In this particular tutorial, we focus on the single dev workflow and go through a full example from Epinio installation to your first app deployment.
+
+## Prerequisites
+
+Before you can use Epinio, you need to have a Kubernetes cluster running.
+
+As a single dev, you might be using a local Kubernetes cluster such as [Rancher Desktop](https://rancherdesktop.io/), [Minikube](https://minikube.sigs.k8s.io/docs/) or [k3d](https://k3d.io/), just to name a few.
+
+:::note
+
+This tutorial will not explain how to install a local Kubernetes cluster and assumes you know how to do it.
+However, depending on your choice, there's already some examples in the **Epinio Howtos** section which contains links to the respective clusters' installation docs.
+
+:::
+
+In this tutorial, we'll use [Rancher Desktop](../howtos/install_epinio_on_rancher_desktop.md) as our local Kubernetes cluster. However you should be able to follow this tutorial with the local Kubernetes cluster of your choice.
+
+If not already done, you can install the [latest version](https://github.com/rancher-sandbox/rancher-desktop/releases) of Rancher Desktop for your operating system.
+
+### [Optional] Additional binaries
+
+Two additional binaries need to be installed in your system:
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) for communicating with the Kubernetes cluster
+
+- [helm](https://helm.sh/docs/intro/install/) for deploying the Epinio Helm Charts
+
+Depending on the local Kubernetes cluster you installed (i.e. Rancher Desktop), these two binaries might be already installed.
+
+## Installation
+
+Once you have your local Kubernetes cluster installed and running, you can [install Epinio](../installation/install_epinio.md).
+
+Here are the steps for Rancher Desktop:
+
+```shell
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
+
+# Wait for cert-manager to stabilize
+
+helm repo add epinio https://epinio.github.io/helm-charts
+
+helm repo update
+
+helm install epinio -n epinio --create-namespace epinio/epinio --set global.domain=127.0.0.1.sslip.io
+```
+
+### CLI
+
+To interact with your Epinio installation, you've to download the [Epinio CLI binary](https://github.com/epinio/epinio/releases/latest). The binary is available for the three main Operating Systems (OS), so pick the one suited for your own OS.
+
+### Login
+
+The first task to perform after Epinio installation, is to [login](../references/commands/cli/epinio_login.md) with the binary you downloaded:
+
+```shell
+epinio login -u admin 'https://epinio.127.0.0.1.sslip.io'
```

Comment:
What's the password? 😃

Reply:
The `NOTES.txt` printed at the end of deploying `epinio` with `helm` tells you.